assets/src/dashboard/components/popoverMenu/menu.js

```diff
@@ -65,6 +65,22 @@ export const MenuItem = styled.li`
     &:focus, &:active, &:hover {
       color: ${isDisabled ? theme.colors.gray400 : theme.colors.gray700};
     }
+    
+    ${
+      separator === 'top'
+        ? `
+        border-top: 1px solid ${theme.colors.gray50};
+      `
+        : ''
```

Comment:
Little nit but we can short circuit this and do
```
separator === 'top' && `border-top: 1px solid ${theme.colors.gray50};`
```

Reply:
I actually had that at first but was bothered by the `false:false` that's rendered as a style when the separator isn't present in the list item. 
<img width="296" alt="Screen Shot 2020-08-31 at 8 55 29 AM" src="https://user-images.githubusercontent.com/10720454/91741353-711c6080-eb69-11ea-8d2d-411e395b3a7f.png">
Instead I decided to take this nit and Dillon's suggestion for a boolean instead of a string and just add a className to the list items that need the border top or bottom so that it's easier to tell what the `item.separator` does inside the map and eliminates the conditional rendering inside the styles. 

Thoughts?